eng/vscode/package.json

```diff
@@ -53,6 +53,17 @@
                     "type": "array",
                     "items": { "type": "string" },
                     "description": "List of Azure MCP services to enable. If empty or unset, all services are enabled by default."
+                },
```

Comment:
Is it possible to change how these items are displayed?

<img width="881" height="320" alt="Image" src="https://github.com/user-attachments/assets/e453bf81-bec7-42e2-b48d-b1bc664ab6c7" />

Reply:
Last time I looked at it the answer was no. VS Code infers the title from the configuration id and there is no way to customize them.

Reply:
yeah, we're keeping the prefix, as it is also helpful when working with settings in raw .json